components/butterfly-test/src/lib.rs

```diff
@@ -20,6 +20,7 @@ extern crate time;
 extern crate habitat_butterfly;
```

Comment:
Don't know how to comment on that but the commit message went way over my head (The first sentence especially)

Is there a way to reformulate it without using the forbidden c-word ("class")?

Reply:
Yeah, sounds crappy. Probably need to figure out the proper rustacean wording here. And probably elaborate a bit more.